pallets/creditcoin/src/migrations/v7.rs

```diff
@@ -20,6 +20,21 @@ pub(crate) fn migrate<T: Config>() -> Weight {
 	crate::weights::WeightInfo::<T>::migration_v7(n)
 }
 
+#[cfg(feature = "try-runtime")]
+pub(crate) fn pre_upgrade<T: Config>() -> Result<(), &'static str> {
+	Ok(())
+}
+
+#[cfg(feature = "try-runtime")]
+pub(crate) fn post_upgrade<T: Config>() -> Result<(), &'static str> {
+	ensure!(
+		StorageVersion::get::<crate::Pallet<T>>() == 7,
```

Comment:
It looks trivial to me, we always bump storage version at the end of a successful migration. Why would it not be 7?
 We will need to modify this line on each storage version bump (i.e. for each migration), should we include a check in CI?

Reply:
> It looks trivial to me, we always bump storage version at the end of a successful migration. Why would it not be 7?

For example a copy& paste error in `migrations/mod.rs` which incorrectly sets storage version. 

IIRC we didn't come up with anything more to assert on here other then the number. The rest of the testing activities will be around making sure migrations are applied without crashing and then forking the blockchain and running the integration test suite against the fork.


> We will need to modify this line on each storage version bump (i.e. for each migration), should we include a check in CI?

Good point. I am not sure how to handle this elegantly though. Will think about it a bit more.

Reply:
@CAGS295 added a new sanity check script, see https://github.com/gluwa/creditcoin/actions/runs/3874453271/jobs/6605673537